dangerfile.js

```diff
@@ -86,11 +86,15 @@ const noFlowFiles = newJsFiles
 
 raiseIssueAboutPaths(warn, noFlowFiles, '@flow');
 
+// based on assumptions from https://github.com/nodejs/node/wiki/ES6-Module-Detection-in-Node
+// basically it needs to contain at least one line with import or export at the
+// beginning of it, followed by a space; any content thereafter doesn't matter
+const esModuleRegex = /^(import|export)\s/g;
```

Comment:
@thymikee Hmm so this is somewhat complicated... since files that are typed by Flow _also_ have the `@flow` comment, it's nearly impossible to differentiate the flow type files from normal ES modules. What do you recommend here, should I just roll back this change?

In general I'm a little confused by the fact that flow uses ES module-like `import` syntax. Does that mean those files need transpilation via Babel?